src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -83,7 +118,9 @@ public String beforeOrAfter(LocalDate someDate) {
      * @return LocalDateTime
      */
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        String dateAndZone = dateInString + "[" + zone + "]";
```

Comment:
let's try to be less straightforward and solve this with the help of Instant class